src/actionCreators/resourceHelpers.js

```diff
@@ -504,7 +504,7 @@ const newUriFromObject = (obj, property, propertyUri, context) => {
   const labelQuads = context.dataset
     .match(obj, rdf.namedNode("http://www.w3.org/2000/01/rdf-schema#label"))
     .toArray()
-  let label = uri
+  let label = null
```

Comment:
No longer defaulting the label for a URI to that URI when a label is not present.